config/monitoring/toolchain_monitoring_namespace.tmpl.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: grafana-serviceaccount-template
```

Comment:
where is this file used?

Reply:
nowhere, it's a left-over after some refactoring :|